include/Model/Languages.php

```diff
@@ -194,22 +194,30 @@ public function add( $args ) {
 		 * } $args
 		 */
 		// First the language taxonomy.
-		$r = wp_insert_term(
+		$result = wp_insert_term(
 			$args['name'],
 			'language',
 			array(
 				'slug'        => $args['slug'],
 				'description' => $this->build_metas( $args ),
 			)
 		);
-		if ( is_wp_error( $r ) ) {
-			// Avoid an ugly fatal error if something went wrong (reported once in the forum).
-			return new WP_Error( 'pll_add_language', __( 'Impossible to add the language.', 'polylang' ) );
+		if ( is_wp_error( $result ) ) {
+			/* translators: %s is the detailed error message */
+			return new WP_Error( 'pll_add_language', __( 'Impossible to add the language, database error: %s.', 'polylang' ), $result->get_error_message() ); // Avoid an ugly fatal error if something went wrong (reported once in the forum).
```

Comment:
Then I guess we should remove **_database_** from the error message too ?

Reply:
👍 
Done in 5570582c.